tomviz/modules/ModuleSlice.cxx

```diff
@@ -679,11 +679,18 @@ void ModuleSlice::onSliceChanged(double* point)
   imageData()->GetDimensions(dims);
   double bounds[6];
   imageData()->GetBounds(bounds);
-  int slice;
 
-  slice = (dims[axis] - 1) * (point[axis] - bounds[2 * axis]) /
+  // Due to changes from https://github.com/OpenChemistry/tomviz/pull/2014
```

Comment:
You should use the commit SHA if anything, but referencing previous commit is fine by me. I don't want to litter our code with URLs. If you want to use the SHA, just use the first 8 characters.